runelite-client/src/main/java/net/runelite/client/plugins/nightmarezone/NightmareZoneOverlay.java

```diff
@@ -67,6 +67,8 @@
 		this.plugin = plugin;
 		this.infoBoxManager = infoBoxManager;
 		this.itemManager = itemManager;
+
+		panelComponent.setPreferredSize(new Dimension(150, 0));
```

Comment:
Instead of making it bigger, just change total points to total and points/hour to Points/Hr